server/src/index.js

```diff
@@ -32,6 +32,12 @@ app.use(errorHandler);
 
 const { port } = config.server;
 
-app.listen(port, () => {
-    console.log(`Server is listening on port ${port}...`);
-});
+di.get('services.data.sample')
```

Comment:
do we need this?

Reply:
@rafal-ksiazek-rmtcfm-com Yes, because we want to load all sample data before server starts. Or not?